elftools/dwarf/dwarfinfo.py

```diff
@@ -290,19 +290,23 @@ def _parse_CU_at_offset(self, offset):
         # with a default address_size=4. If, after parsing the header, we
         # find out address_size is actually 8, we just create a new structs
         # object for this CU.
+        # Same logic for DWARF version. Some structs, notably DW_FORM_ref_addr,
```

Comment:
This whole comment block is out of date now.

I propose to just unconditionally create a new `cu_structs` - it's cheap. And modify the comment accordingly. WDYT?

Reply:
Made the structs creation unconditional, explained in the comments.